tests/mock_network/node_ageing.rs

```diff
@@ -150,6 +151,68 @@ fn relocate_during_split() {
     })
 }
 
+// Test that during startup phase all churn events cause age increments.
+#[test]
+fn startup_phase() {
+    let env = Environment::new(NETWORK_PARAMS);
+    let mut nodes = vec![];
+
+    // Only the first `safe_section_size - 1` adds cause age increments, the rest does not.
+    for i in 0..(env.safe_section_size() + 1) {
```

Comment:
the comment says `safe_section_size - 1` but the code is `safe_section_size() + 1`
I thought the code shall be `0..safe_section_size()` ?

Reply:
We iterate safe_section_size + 1 times to test both cases: while we are below safe_section_size, the ages are incremented, but at and past it they are not.